src/main/java/com/jayway/maven/plugins/android/phase_prebuild/ClasspathModifierLifecycleParticipant.java

```diff
@@ -265,8 +265,15 @@ private void addClassesToClasspath( UnpackedLibHelper helper, MavenProject proje
             throw new MavenExecutionException( "Could not add " + classesJar.getName() + " as dependency", e );
         }
 
-        // Add the classes to the classpath
-        final Dependency dependency = createSystemScopeDependency( artifact, classesJar, "extracted" );
+        
+        // Modify the classpath to use an extracted dex file.  This will overwrite
```

Comment:
It's not using an extracted dex file though is it?
It's using the classes.jar from an unpacked aar lib.